packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -72,8 +72,11 @@ export const defaultTablesConfig: TablesConfig = {
     },
     {
       "id": "actions",
-      "columnType": "extension-column",
-      "header": "Actions"
+      "columnType": "actions-column"
+    },
+    {
+      "id": "active-visit-actions",
+      "columnType": "extension-column"
     }
   ],
   "tableDefinitions": [
```

Comment:
To enable the current actions (bell icon + Transfer + overflow menu actions to  Edit patient + End visit). Copy this configuration entry and replace `"actions"` with `"active-visit-actions"` in the `columns` array in line 84.